bin/action_subscriber

```diff
@@ -39,23 +40,16 @@ module ActionSubscriber
     end
   end
 
-  [:INT, :QUIT, :TERM].each do |signal|
-    trap(signal) do
-      puts "Stopping server..."
-      wait_loops = 0
+  if ::RUBY_PLATFORM == "java"
```

Comment:
not understanding the split here .... in my experience jruby will catch TERM, QUIT, INT in most cases ..... of course if `stop_server!` is reentrant then we can register both instead of a separate line per platform